src/config/utils.ts

```diff
@@ -19,16 +20,23 @@ type tableKeys = keyof typeof rawTables;
 const getLayerByKey = (layerKey: layerKeys): LayerType => {
   const rawDefinition = rawLayers[layerKey];
 
-  const definition = {
+  const definition: { id: layerKeys; type: LayerType['type'] } = {
     id: layerKey,
-    type: rawDefinition.type,
+    type: rawDefinition.type as LayerType['type'],
     ...mapKeys(rawDefinition, (v, k) => camelCase(k)),
   };
 
   const throwInvalidLayer = () => {
     throw new Error(`Found invalid layer definition for layer '${layerKey}'.`);
   };
 
+  const ll: 'dd' | 'ww' = 'dd';
```

Comment:
👣 ?

Reply:
Looks like I was experimenting with types there when trying to enforce `LayerType`, good catch!

I'll try be a little more careful next time.